makefile

```diff
@@ -9,21 +9,29 @@ US_vaccination_search_insights.csv: $(SRC_DIR)
 GB_vaccination_search_insights.csv: $(SRC_DIR)
 	curl -o $(SRC_DIR)/GB_vaccination_search_insights.csv https://storage.googleapis.com/covid19-open-data/covid19-vaccination-search-insights/GB_vaccination_search_insights.csv
 
+CA_vaccination_search_insights.csv: $(SRC_DIR)
+	curl -o $(SRC_DIR)/CA_temp.csv https://storage.googleapis.com/covid19-open-data/covid19-vaccination-search-insights/CA_vaccination_search_insights.csv
+	cat $(SRC_DIR)/CA_temp.csv | awk 'NR==1' > $(SRC_DIR)/$@
+	cat $(SRC_DIR)/CA_temp.csv | awk -vFPAT='[^,]*|"[^"]*"' -v OFS=',' 'NR>1 { print $$1,$$2,$$3,$$4,$$5,$$9,$$9,"","",$$10,$$11,$$12,$$13 }' >> $(SRC_DIR)/$@
```

Comment:
Don't modify the CSV format here, any issues with the format should to be fixed on the backend.

Reply:
It's not really issues. It makes the front-end implementation much easier. We're treating FSAs as sub_region_2 in the front-end. But they should be sub_region_3 in the back-end. We could change it, but for simplicity of things and not making CA-specific logic, we decided to go this route